docs/en/docs/sql-manual/sql-reference/Data-Manipulation-Statements/Manipulation/SELECT.md

```diff
@@ -65,26 +65,28 @@ SELECT
    3. where_definition retrieves the condition (expression), if there is a WHERE clause, the condition filters the row data. where_condition is an expression that evaluates to true for each row to be selected. Without the WHERE clause, the statement selects all rows. In WHERE expressions, you can use any MySQL supported functions and operators except aggregate functions
 
    4. `ALL | DISTINCT ` : to refresh the result set, all is all, distinct/distinctrow will refresh the duplicate columns, the default is all
+   
+   5. `ALL EXCEPT`: Filter on the full (all) result set, except specifies the name of one or more columns to be excluded from the full result set. All matching column names will be ignored in the output.
```

Comment:
```suggestion
   5.  <version since="1.2" type="inline"> `ALL EXCEPT`: Filter on the full (all) result set, except specifies the name of one or more columns to be excluded from the full result set. All matching column names will be ignored in the output. </version>
```

Reply:
See: https://doris.apache.org/zh-CN/community/how-to-contribute/contribute-doc/#%E8%A1%8C%E5%86%85%E6%A0%87%E7%AD%BE

Reply:
Done